src/datasources/location-data-source.ts

```diff
@@ -6,13 +6,22 @@ export class LocationDataSource extends BaseOpenMRSDataSource {
     super('/ws/rest/v1/location?v=custom:(uuid,display)');
   }
 
-  fetchData(searchTerm: string, config?: Record<string, any>): Promise<any[]> {
-    if (config?.tag) {
-      let urlParts = this.url.split('?');
-      this.url = `${urlParts[0]}?tag=${config.tag}&${urlParts[1]}`;
+  fetchData(searchTerm: string, config?: Record<string, any>, uuid?: string): Promise<any[]> {
+    let apiUrl = this.url;
```

Comment:
was the intention of this to support external links outside the openmrs ecosystem?